api/helpers/gamestate/check-game-state-for-win.js

```diff
@@ -16,9 +16,14 @@ module.exports = {
       description: 'GameState object',
       required: true,
     },
+    countPasses: {
+      type: 'ref',
```

Comment:
If we immediately invoke, this could be type of number

Reply:
@itsalaidbacklife  @Haviles04  how about this way.
``` 
      const countPasses = () => {
        let numPasses = 0;
        for (const gameState of game.gameStates.slice(-3)) {
          if (gameState.moveType !== MoveType.PASS) {
            return numPasses;
          }
          numPasses++;
        }
        return numPasses;
      };

      **const numPasses = countPasses();**

      const victory = await sails.helpers.gamestate.checkGameStateForWin(game, gameState, numPasses);
```